lstchain/mc/sensitivity.py

```diff
@@ -5,11 +5,15 @@
 from .mc import rate, weight
 from lstchain.spectra.crab import crab_hegra,crab_magic
 from lstchain.spectra.proton import proton_bess
-from gammapy.stats import WStatCountsStatistic
 from lstchain.reco.utils import reco_source_position_sky
 from astropy.coordinates.angle_utilities import angular_separation
 from lstchain.io import read_simu_info_merged_hdf5
 from lstchain.io.io import dl2_params_lstcam_key
+try:
+    from gammapy.stats import WStatCountsStatistic
+except ImportError:
+    from gammapy.stats import wstat as WStatCountsStatistic
```

Comment:
where is this import error coming from? I tested it with gammapy 0.17 and did not get it

Reply:
Yes, please just update gammapy using `conda install -c conda-forge gammapy=0.17`

Reply:
My bad, I had gammapy 0.16 installed, with gammapy 0.17 it works. I will revert the change and add 0.17 as a minimum version in setup.py

Reply:
I already did in #333